docs/content/community/contributing.mdx

```diff
@@ -9,6 +9,53 @@ We love to see our community members get involved! If you are planning to contri
 
 ## Environment Setup
 
+### Apple M1 specific setup
+
+If you have a Mac with an M1 chip, you will need to set up a separate environment to emulate an x86 architecture before following the remaining setup instructions.
+
+1. Install rosetta
```

Comment:
You can merge without this, but it would be helpful to include a link to Rosetta so people know what it is.